keyboards/grumpy_sloth/slim_sloth/info.json

```diff
@@ -0,0 +1,143 @@
+{
+    "manufacturer": "Grumpy Sloth",
+    "keyboard_name": "Slim Sloth",
+    "maintainer": "RonnyM82",
+    "bootloader": "rp2040",
+    "debounce": 5,
```

Comment:
```suggestion
```

Reply:
@waffle87 what's the reason for debounce being removed here please? I don't see it being replaced anywhere else.

Reply:
Is it because 5 is the default value anyway?